airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -80,7 +86,7 @@ class JenkinsJobTriggerOperator(BaseOperator):
     :param job_name: The name of the job to trigger
     :type job_name: str
     :param parameters: The parameters block to provide to jenkins. (templated)
-    :type parameters: str
+    :type parameters: JSON blob, path to JSON file, Dict, or List
```

Comment:
```
    :param parameters: The parameters block to provide to jenkins. JSON blob, path to JSON file, Dict, or List. (templated)
    :type parameters: Union[str, Dict, List]
```
I think the type should be explicit, WDYT?

Reply:
You mean essentially just put the exact type like you have here? That seems like wider practice, right? I can make that change.

Reply:
Yes, and you can add the information about json etc to `:param` part so users understand what the type mean

Reply:
@turbaszek - gave this a shot in the latest commit. Let me know what you think.